src/modules/popup.js

```diff
@@ -0,0 +1,54 @@
+const showPopup = (movie) => {
+  const projectModal = document.querySelector('.popup');
+  const body = document.querySelector('body');
+  projectModal.style.display = 'block';
+  const modalVisible = `
+              <div class="popup-modal">
+              <div class="image-close flex-center">
+                  <div class="modal-image">
+                      <img src="${movie.image.medium}" alt="">
```

Comment:
- [ ] Please add a value for the `alt` of the `img` to make the content more accessible. I will suggest 
```suggestion
                      <img src="${movie.image.medium}" alt="Image of ${movie.name} movie">
```